charts/harvester-csi-driver/100.0.0+up0.1.9/Chart.yaml

```diff
@@ -1,10 +1,10 @@
 annotations:
   catalog.cattle.io/certified: rancher
   catalog.cattle.io/display-name: Harvester CSI Driver
-  catalog.cattle.io/kube-version: '>= 1.18'
+  catalog.cattle.io/kube-version: '>= 1.18.0-0 < 1.25.0-0'
```

Comment:
same comment as above here, not sure if this initial chart was tested on 1.24 https://github.com/rancher/charts/pull/1925#discussion_r903262223

Reply:
updated